dojo/settings/settings.dist.py

```diff
@@ -1456,6 +1457,7 @@ def saml2_attrib_map_format(dict):
     'Whispers': DEDUPE_ALGO_HASH_CODE,
     'Blackduck Hub Scan': DEDUPE_ALGO_HASH_CODE,
     'BlackDuck API': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL,
+    'Blackduck Binary Analysis': DEDUPE_ALGO_HASH_CODE,
```

Comment:
```suggestion
    'Blackduck Binary Analysis': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL_OR_HASH_CODE,
```

Reply:
Changed in favor of relying upon updated `unique_id_from_tool`.